backend/route/TeacherRoute.js

```diff
@@ -1,2 +1,33 @@
 import express from "express";
+import { boolean } from "yargs";
```

Comment:
This Is unused Library!!
Please Remove this.

Reply:
Ok